entities/dashboard/staffWidgetsBoard.json

```diff
@@ -0,0 +1,115 @@
+{
+    "type": "object",
+    "properties": {
+      "uid": {
+        "type": "string",
+        "description": "Unique identifier of the staffWidgetsBoard object"
+      },
+      "board_layout": {
+        "type": "string",
+        "description": "The CSS class of the layout. This will determine the layout of the widgets dashboard based on the class.",
+        "enum": [
+          "dashboard"
+        ]
+      },
+      "type": {
+        "type": "string",
```

Comment:
See my comment. this is not the purpose of the property type. See Eldad's HLD. It supposed to define the page this board belongs to (like home). Also, you should enforce unique check for staff_uid and type as a staff can have one for home and one for payments. This should also be enabled with the GET List (enable type as a filter)

Reply:
This what was discussed and approved by views team, name and values.